config/betterquesting/DefaultQuests.json

```diff
@@ -22591,7 +22315,7 @@
           "ismain:1": 0,
           "repeat_relative:1": 1,
           "name:8": "Acetone",
-          "desc:8": "There are 2 main options for Acetone:\n\nOption 1 - Distill Wood Vinegar (from pyrolyse).\n\nOption 2 - Make Acetic Acid first, then react with Calcium (or a composite with Calcium, such as Calcite or Quicklime) for Calcium Acetate Solution. Distill or heat that up for Acetone. Using a Distillation Tower for the last step gives back Quicklime.\n\nOption 3 - As a byproduct from making Phenol from Cumene (check that quest out)."
+          "desc:8": "Asides from Epoxy, Acetone can be used to refine Diesel or make Gasoline. The different ways to obtain it are:\n\nOption 1 - Distill Wood Vinegar (from pyrolyse).\n\nOption 2 - As a byproduct from making Phenol from Cumene (check that quest out).\n\nOption 3 - Make Acetic Acid first, then react with Calcite or Quicklime for Calcium Acetate Solution. Distill or heat that up for Acetone. The Calcium compound is lost, unless you use a Distillation Tower (available at EV tier) for the final step."
```

Comment:
What do you mean "refine Diesel"? I can't find any use of acetone for diesel, unless you mean Methyl Acetate produced with Acetic Acid.

Reply:
Sorry, gt5 brain. I probably wanted an introduction to the chemical but this was such a poor attempt. After a little bit of googling (because I'm not an expert), I went with "Acetone is one out of many ketones, also the simplest.".